sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -698,6 +698,8 @@ private[sql] class SQLConf extends Serializable with CatalystConf with Logging {
   override def orderByOrdinal: Boolean = getConf(ORDER_BY_ORDINAL)
 
   override def groupByOrdinal: Boolean = getConf(GROUP_BY_ORDINAL)
+
+  override def allowCartesianProduct: Boolean = getConf(CROSS_JOINS_ENABLED)
```

Comment:
As discussed, I only have the crossJoinEnabled flag, which has been exposed in CatalystConf. Also as discussed, deprecating crossJoinEnabled and switching the conf flag to requireCrossJoinSyntax is a good idea, will address in a follow-up PR.